src/commcare_cloud/commands/secrets.py

```diff
@@ -47,7 +47,11 @@ def _secrets_edit(self, environment, secret_name):
         try:
             secret_value = json.loads(secret_value)
         except ValueError:
-            pass
+            puts(color_error(
+                'Could not understand value, please use valid json: {value}'
+                .format(value=secret_value)
+            ))
+            exit(-1)
```

Comment:
Yeah this is a bit of a blemish, but I wanted to accept here either JSON if it parses that way or else a string. For string values we to be able to type in `****myspecialsecret****` and not `"****myspecialsecret****"`, but we also want a way to input JSON.

That said If we're fine just making people type in `"****myspecialsecret****"` for a string secret, then this change will do that. Perhaps if we go that route we could be kind to that person and add something like 'If this was a string, did you mean to put it in double quotes ("...")?'. I'd say most of these secrets are strings

Reply:
I see your point. When most secrets are strings, having to add quotes does seems pretty annoying. What if I grabb the current value, and only raise an error here if the current value is a list/dict **and** the new secret doesn't parse as json? It wouldn't be perfect - for new values, I'd just assume the value should be a string, but that could be incorrect - but it'd be better.

Reply:
Yeah I suppose that would work!

Reply:
de9cf8e2a978ec610f44b1f0e18115aa0b2880b0